packages/react-components/react-combobox/docs/Spec.md

```diff
@@ -22,20 +24,23 @@ Select provides better mobile support and accessibility than Combobox, and has t
 
 Use Select when a basic single-select form component with no freeform text input or filtering is needed.
 
-### When to use Combobox
+### When to use Combobox or Dropdown
 
-Combobox is a more feature-rich version of Select, which comes at the cost a larger code footprint, and less robust support for accessibility compared to the native `<select>` element.
+Dropdown is a more feature-rich version of Select, which comes at the cost a larger code footprint, and less robust support for accessibility compared to the native `<select>` element. Combobox (as mentioned previously) is essentially a Dropdown that allows text input.
```

Comment:
I'd drop the "(as mentioned previously)" paranthetical. Documentation isn't always read top to bottom so it's reasonable that a read won't have seen this previous mention.